tests/Metadata/Resource/Factory/CachedResourceMetadataFactoryTest.php

```diff
@@ -86,6 +86,6 @@ public function testCreateWithGetCacheItemThrowsCacheException()
 
     private function generateCacheKey(string $resourceClass = Dummy::class)
     {
-        return CachedResourceMetadataFactory::CACHE_KEY_PREFIX.md5(serialize([$resourceClass]));
```

Comment:
@teohhanhui you had a good reason to serialize here? (I think it comes from you)

Do we really need the `md5` thing?